src/utils/finance/source.py

```diff
@@ -0,0 +1,266 @@
+                raw values in the enum column to their standard values
+            column_to_date_format: dict mapping column names to their date format
+        """
+        if enum_mapper is not None:
+            self.enum_mapper = enum_mapper
+        if column_to_date_format is not None:
+            self.column_to_date_format = column_to_date_format
+        standard_schema_table = self._standardize_enums(standard_schema_table)
+        standard_data_table = self._standardize_column_to_date_format(
+            standard_schema_table
+        )
+        return standard_data_table
+
+
+class DataSourceStandardizationPipeline:
+    """Run pipeline for a single data source from file to standardized dataframe"""
+
+    @property
+    def table_type(self) -> str:
+        """Table type that pipeline standardizes"""
+        return self.config_handler.table_type
+
+    def __init__(
+        self,
+        state_code: str,
+        form_code: str,
+        config_file: Path = None,
+        data_reader: DataReader = None,
+        schema_transformer: SchemaTransformer = None,
+        data_standardizer: DataStandardizer = None,
+    ) -> None:
+        """Initialize data source standardization for a single data source
+
+        Args:
+            state_code: two letter abbreviation for state
+            form_code: code to identify data source form type in state config file
+            config_file: Path to config file. If not provided, state's default
+                config file in config/finance/{state_code}.yaml will be used
+            data_reader: logic for reading data into raw dataframe. If none provided
+                will use default logic with provided config
+            schema_transformer: logic for transforming schema of raw data. If none
+                provided, will use default logic with provided config
+            data_standardizer: logic for standardizing data of the raw file. If
+                none provided, will use default logic with provided config.
+        """
+        self.state_code = state_code.lower()
+        self.form_code = form_code
+        if config_file is not None:
+            self.config_handler = ConfigHandler(form_code, config_file_path=config_file)
+        else:
+            self.config_handler = ConfigHandler(form_code, state_code=self.state_code)
+        if data_reader is None:
+            self.data_reader = DataReader(self.config_handler)
+        else:
+            self.data_reader = data_reader
+        if schema_transformer is None:
+            self.schema_transformer = SchemaTransformer(self.config_handler)
+        else:
+            self.schema_transformer = schema_transformer
+        if data_standardizer is None:
+            self.data_standardizer = DataStandardizer(self.config_handler)
+        else:
+            self.data_standardizer = data_standardizer
+
+    def standardize_data_source(self) -> pd.DataFrame:
+        """Process all data for a data_source"""
```

Comment:
nit: I think this docstring should probably specify that it's returned in a single concatenated dataframe.